MonoGame.Framework/Utilities/InteropHelpers.cs

```diff
@@ -0,0 +1,33 @@
+// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace MonoGame.Utilities
+{
+    internal static class InteropHelpers
+    {
+        // Convert a pointer to a Utf8 null-terminated string to a .NET System.String
```

Comment:
If you want to add a comment for what the method does, make it a documentation comment so its seen from intellisense.